Tasks/AzureRmWebAppDeploymentV3/operations/TaskParameters.ts

```diff
@@ -52,6 +53,19 @@ export class TaskParametersUtility {
             taskParameters.AdditionalArguments = tl.getInput('AdditionalArguments', false);
         }
 
+        if(taskParameters.isLinuxApp && taskParameters.ScriptType) {
```

Comment:
Can you please make this is one method like ScriptypeLinuxParameter parsing to make it modular ?